pkg/controller/provider/web/ova/workload.go

```diff
@@ -0,0 +1,91 @@
+package ova
+
+import (
+	"errors"
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+	api "github.com/konveyor/forklift-controller/pkg/apis/forklift/v1beta1"
+	model "github.com/konveyor/forklift-controller/pkg/controller/provider/model/ova"
+	"github.com/konveyor/forklift-controller/pkg/controller/provider/web/base"
+)
+
+// Routes.
+const (
+	WorkloadCollection = "workloads"
+	WorkloadsRoot      = ProviderRoot + "/" + WorkloadCollection
+	WorkloadRoot       = WorkloadsRoot + "/:" + VMParam
+)
+
+// Virtual Machine handler.
+type WorkloadHandler struct {
+	Handler
+}
+
+// Add routes to the `gin` router.
+func (h *WorkloadHandler) AddRoutes(e *gin.Engine) {
+	e.GET(WorkloadRoot, h.Get)
+}
+
+// List resources in a REST collection.
+func (h WorkloadHandler) List(ctx *gin.Context) {
```

Comment:
Sonarcloud: `Add a nested comment explaining why this function is empty or complete the implementation.`